src/System.Xml.XmlDocument/tests/project.lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "locked": true,
+  "locked": false,
```

Comment:
Not intended, no idea what this means, just got auto changed when I built so I decided to add this to review